src/directory/directory.mjs

```diff
@@ -1103,11 +1103,51 @@ export const directory = {
           ]
         },
         {
-          path: 'src/pages/[platform]/reference/index.mdx',
-          children: [
-            { path: 'src/pages/[platform]/reference/flutter-api/index.mdx' }
+          isExternal: true,
+          route: 'https://aws-amplify.github.io/amplify-js/api/',
```

Comment:
Think we should kill this as it's just relevant for JS v5, there is no and won't be a v6 version.

Reply:
We can but I've hidden it for v6 in this PR: https://next-release-references.d1egzztxsxq9xz.amplifyapp.com/